File: Selenium/Activities/Activity15.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity15 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		
		driver.get("https://training-support.net/webelements/dynamic-attributes");
		System.out.println("Title of the page: " +driver.getTitle());
		
		WebElement fullName = driver.findElement(By.cssSelector("input[id^='full-name']"));
        WebElement email = driver.findElement(By.cssSelector("input[id$='-email']"));
        WebElement eventDate = driver.findElement(By.xpath("//input[contains(@name, '-event-date-')]"));
        WebElement details = driver.findElement(By.xpath("//textarea[contains(@id, '-additional-details-')]"));
        WebElement submit = driver.findElement(By.xpath("//button[text()='Submit']"));
        
        fullName.sendKeys("abc");
        email.sendKeys("abc@xyz.com");
        eventDate.sendKeys("2025-01-26");
        details.sendKeys("Details");
        submit.click();
        
        String result = driver.findElement(By.id("action-confirmation")).getText();
        System.out.println("Result: " +result);
        
        driver.quit();        
	}

}

File: Selenium/Activities/Activity15.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-attributes")
    print(f"Title of the page: {driver.title}")

    full_name = driver.find_element(By.CSS_SELECTOR, "input[id^='full-name']")
    email = driver.find_element(By.CSS_SELECTOR, "input[id$='-email']")
    event_date = driver.find_element(By.XPATH, "//input[contains(@name, '-event-date-')]")
    details = driver.find_element(By.XPATH, "//textarea[contains(@id, '-additional-details-')]")
    submit = driver.find_element(By.XPATH, "//button[text()='Submit']")

    full_name.send_keys("abc")
    email.send_keys("abc@xyz.com")
    event_date.send_keys("2025-01-26")
    details.send_keys("Details")
    submit.click()
        
    result = driver.find_element(By.id("action-confirmation")).text
    print("Result: " +result)
        